TransferMethod/Sources/View/Widget/TextWidget.swift

```diff
@@ -36,6 +36,11 @@ class TextWidget: AbstractWidget {
         return String(String.UnicodeScalarView(numbers.compactMap(UnicodeScalar.init)))
     }
 
+    private var allowedSpecialCharacters: String {
+         let specialCharacters = UInt32("(") ... UInt32(")")
```

Comment:
Couple of issues with this logic
- Code in this method is overkill for simply getting `"()"` as there are no other characters between `UInt32("(")` and` UInt32(")")`
- This logic specifically deals with `"()"` characters only and so issue will still exist for other special characters like `"{}$%^"` etc..

See `po` output
```
(lldb) po allowedLetters
"abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"

(lldb) po allowedNumbers
"0123456789"

(lldb) po allowedSpecialCharacters
"()"
```